src/index.scss

```diff
@@ -1,4 +1,16 @@
 @import 'styles/tailwind';
+@import 'styles/_fonts';
+
+:root {
```

Comment:
We could move this into 'styles/_colors';

Reply:
Yes agree let's have an additional `src/style` folder!

Reply:
Or would you rather add it in the root folder?

Reply:
Yeah already added the src/styles, will jsut create another file for the colors to put them in there as variables 
I'd leave the index here though next to the other for now